dev/tools/localization/bin/gen_localizations.dart

```diff
@@ -381,16 +381,21 @@ $factoryDeclaration
 /// Typically "String", but some (e.g. "timeOfDayFormat") return enums.
 ///
 /// Used by [generateGetter] below.
-String generateType(Map<String, dynamic> attributes) {
+String generateType(String key, Map<String, dynamic> attributes) {
```

Comment:
Where is `key` being used here?

Reply:
Good catch. Originally I thought I might need to special case them off the key name, but then found a better way. This was left over. I just removed it. Thx.